app/views/searches/_form.html.erb

```diff
@@ -1,6 +1,14 @@
 <%= content_tag :section, id: dom_id(search) do %>
   <div class="py-8 px-4 max-w-2xl lg:py-16">
     <%= form_with model: search do |f| %>
+      <%= render Forms::ToggleComponent.new(form: f, name: :test2, label: 'Toggle me', value: true) %>
+
+      <%= f.label :test, class: 'relative inline-flex items-center cursor-pointer' do %>
+        <%= f.check_box :test, class: 'sr-only peer' %>
+        <div class="w-11 h-6 bg-gray-200 peer-focus:outline-none peer-focus:ring-4 peer-focus:ring-blue-300 rounded-full peer peer-checked:after:translate-x-full peer-checked:after:border-white after:content-[''] after:absolute after:top-[2px] after:left-[2px] after:bg-white after:border-gray-300 after:border after:rounded-full after:h-5 after:w-5 after:transition-all peer-checked:bg-blue-600"></div>
+        <%= content_tag :span, 'Toggle me', class: 'ml-3 text-sm font-medium text-gray-900' %>
+      <% end %>
+
```

Comment:
This is only for test. When I render toggle with view component, then it is not working. When I render toggle in partial, then everything is fine.